docs/pages/guides/web-performance.md

```diff
@@ -28,6 +28,17 @@ npx expo-optimize
 
 To inspect bundle sizes, you can use a Webpack plugin called [_Webpack Bundle Analyzer_](https://github.com/webpack-contrib/webpack-bundle-analyzer). This plugin will help you visualize the size of your static bundles. You can use this to identify unwanted large packages that you may not have bundled intentionally.
 
+
+### Using Bundle Analyzer for app bundle
```

Comment:
This appears to be related to native JS bundle size. The contents of this PR should probably go in a different doc seeing as this one is web specific.

Reply:
I think so too. Which section would be a good fit?

Reply:
https://docs.expo.io/guides/customizing-metro/